usaspending_api/database_scripts/matview_generator/chunked_matview_sql_generator.py

```diff
@@ -159,6 +162,7 @@ def create_componentized_files(sql_json):
 
 def create_chunked_componentized_files(sql_json):
     table_name = sql_json["final_name"]
+    matview_schema_name = sql_json.get("chunked_schema_name", "public")
```

Comment:
I can have a verbal convo to settle this:

I see this with a default of "public". Is that convention everywhere -- fall back to `public` when not found to be set?

And then I see `"chunked_schema"` set like:
```
{
  "schema_name": "rpt",
  "chunked_schema_name": "temp",
```

So the net effect is that the chunked matviews will be created in the `temp` table schema, yes?

Reply:
> Is that convention everywhere -- fall back to public when not found to be set?
The matviews all had that as a default previously so that is why I carried it forward.

> So the net effect is that the chunked matviews will be created in the temp table schema, yes?
That is correct.